install/docker/CUDA/Dockerfile

```diff
@@ -5,10 +5,12 @@ ARG hipsycl_origin=https://github.com/illuhad/hipSYCL
 ENV hipsycl_branch=$hipsycl_branch
 ENV hipsycl_origin=$hipsycl_origin
 RUN apt-get update
-RUN apt-get install -y wget
+RUN apt-get install -y apt-transport-https ca-certificates gnupg software-properties-common wget
 RUN echo "deb http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main" >> /etc/apt/sources.list.d/llvm.list
 RUN echo "deb-src http://apt.llvm.org/bionic/ llvm-toolchain-bionic-8 main" >> /etc/apt/sources.list.d/llvm.list
 RUN wget -O - https://apt.llvm.org/llvm-snapshot.gpg.key | apt-key add -
+RUN wget -O - https://apt.kitware.com/keys/kitware-archive-latest.asc 2>/dev/null | gpg --dearmor - | tee /etc/apt/trusted.gpg.d/kitware.gpg >/dev/null
+RUN apt-add-repository 'deb https://apt.kitware.com/ubuntu/ bionic main'
```

Comment:
okay.. these dockerfiles are deprecated anyway, so no need to put work into it. While it doesn't hurt keeping them up to date, they will probably be removed entirely soonish.

Reply:
No probs, just tried to catch what I saw. Any other workflow files you run that need updates?

Reply:
no updates for linux needed anymore.